subprojects/wrapper/wrapper.gradle

```diff
@@ -26,18 +26,9 @@ dependencies {
     integTestRuntime rootProject.configurations.testRuntime.allDependencies
 }
 
-ext {
-    wrapperBuildReceipt = "build-receipt.properties"
-}
-
-task wrapperBuildReceiptResource() {
-    inputs.property 'version', { version }
-    outputs.file { "${generatedTestResourcesDir}/${wrapperBuildReceipt}" }
-    outputs.cacheIf { true }
-    doLast {
-        generatedTestResourcesDir.mkdirs()
-        new File(generatedTestResourcesDir, wrapperBuildReceipt).text = "versionNumber=${version}"
-    }
+task wrapperBuildReceiptResource(type: Copy) {
+    into generatedTestResourcesDir
+    from project(':versionInfo').createTestBuildReceipt
```

Comment:
The wrapper only needs the version number. The change that we only write the version number in there was introduced by me in https://github.com/gradle/gradle/commit/0d36b76ae4294dbd507f067506ba489e432b040d. Do you think we should create a new file only containing the version number as we did before - this would be closer to what the [production code](https://github.com/gradle/gradle/commit/0d36b76ae4294dbd507f067506ba489e432b040d#diff-b1ccc513de58c872d77e22a19a931a4dR164) is doing right now. On the other hand we could have the production code to produce a full build receipt on creating the wrapper jar. WDYT?
It doesn't contain necessarily new information as before. It is necessary for it to contain the same version as the Gradle distribution we are testing - and this is the version number from the test build receipt.